content/300-guides/400-migrate-to-prisma/02-migrate-from-sequelize.mdx

```diff
@@ -0,0 +1,1217 @@
+  updatedAt DateTime  @updatedAt
+  Posts     Posts[]
+  Profiles  Profiles?
+}
+```
+
+### 2.4. Adjust the Prisma schema (optional)
+
+The models that were generated via introspection currently _exactly_ map to your database tables. In this section, you'll learn how you can adjust the naming of the Prisma models to adhere to [Prisma's naming conventions](../../reference/api-reference/prisma-schema-reference#naming-conventions).
+
+All of these adjustment are entirely optional and you are free to skip to the next step already if you don't want to adjust anything for now. You can go back and make the adjustments at any later point.
+
+As opposed to the current snake_case notation of Prisma models, Prisma's naming conventions are:
+
+- PascalCase for model names
+- camelCase for field names
+
+You can adjust the naming by _mapping_ the Prisma model and field names to the existing table and column names in the underlying database using `@@map` and `@map`.
+
+Also note that you can rename [relation fields](../../concepts/components/prisma-schema/relations#relation-fields) to optimize the Prisma Client API that you'll use later to send queries to your database. For example, the `post` field on the `user` model is a _list_, so a better name for this field would be `posts` to indicate that it's plural. 
+
+Here's an adjusted version of the Prisma schema that addresses these points:
+
+```prisma file=prisma/schema.prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Category {
+  id             Int                @id @default(autoincrement())
+  name           String
+  createdAt      DateTime           @default(now())
+  updatedAt      DateTime           @updatedAt
+  PostCategories PostToCategories[]
+
+  @@map("Categories")
+}
+
+model PostToCategories {
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @updatedAt
+  CategoryId Int
+  PostId     Int
+  Categories Category @relation(fields: [CategoryId], references: [id])
+  posts      Post     @relation(fields: [PostId], references: [id])
+
+  @@id([CategoryId, PostId])
+  @@map("PostCategories")
+}
+
+model Post {
+  id               Int                @id @default(autoincrement())
+  title            String
+  content          String?
+  published        Boolean?           @default(false)
+  createdAt        DateTime           @default(now())
+  updatedAt        DateTime           @updatedAt
+  authorId         Int?
+  author           User?              @relation(fields: [authorId], references: [id])
+  PostToCategories PostToCategories[]
+
+  @@map("Posts")
+}
+
+model Profile {
+  id        Int      @id @default(autoincrement())
+  bio       String
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  userId    Int?
+  user      User?    @relation(fields: [userId], references: [id])
+
+  @@map("Profiles")
+}
+
+model SequelizeMeta {
+  name String @id
+}
```

Comment:
this model seems like it can be removed since it's used internally by Sequelize? I'd make a note in the tutorial that this can be manually deleted from the schema.